ansible-ipi-install/roles/installer/tasks/24_rhcos_image_cache.yml

```diff
@@ -66,6 +66,13 @@
     clusterosimage: "http://{{ ansible_provisioning.ipv4.address }}:8080/{{ rhcos_uri }}?sha256={{ rhcos_sha256 }}"
   when: clusterosimage is not defined or clusterosimage|length < 1
 
+- name: Install required packages
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "podman"
+  become: yes
+
```

Comment:
I want to move this into node-prep/tasks/30_req_packages to keep it easy to figure out where we would find this info.

under existing task in 30_req_packages lets add
```
- name: Install required packages
  package:
    name: "{{ item }}"
    state: present
  with_items: "podman"
  become: yes
  when: cache_enabled == True
```

Reply:
Done